lib/src/oauth2_flows/implicit.dart

```diff
@@ -68,20 +84,24 @@ class ImplicitFlow {
         }
       } catch (error, stack) {
         completer.completeError(error, stack);
+      } finally {
+        _pendingInitialization = null;
```

Comment:
Why set it `null` here?

Reply:
All of the failure cases need to remove `_pendingInitialization`, so another caller could try to load the lib again later if the issue is transient. Otherwise, the future won't return and other calls will hang.

I guess in the case of success, we could keep the future around to prevent all future initializations, not just ones happening at the same time. That's a good point, I've updated the code to remove this from `finally` and only set to `null` in the error block.

Reply:
Spot on :)